template.yaml

```diff
@@ -964,6 +966,33 @@ Resources:
                   - Name: suffix
                     Value: '.json'
 
+  BrageMergingRollbackFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: brage-import
+      Handler: no.sikt.nva.brage.migration.lambda.BrageMergingRollbackHandler::handleRequest
+      Role: !GetAtt LambdaRole.Arn
+      Environment:
+        Variables:
+          BRAGE_MIGRATION_ERROR_BUCKET_NAME: !Sub "${BrageMigrationErrorBucketName}-${AWS::AccountId}"
+          TABLE_NAME: !Ref NvaResourcesTable
+          NVA_PERSISTED_STORAGE_BUCKET_NAME: !Ref ResourceStorageBucketName
+          DOMAIN_NAME: !Ref ApiDomain
+          EVENT_BUS_NAME: !GetAtt InternalBus.Name
+      Events:
+        EventBridgeEvent:
+          Type: EventBridgeRule
+          Properties:
+            EventBusName: !GetAtt InternalBus.Name
+            Pattern:
+              detail:
+                topic: [ "BrageMerging.Rollback.Request" ]
+      EventInvokeConfig:
+        DestinationConfig:
+          OnFailure:
+            Type: SQS
+            Destination: !GetAtt BrageMergingRollbackHandlerDLQ.Arn
```

Comment:
DLQ vil kun benyttes for ting som tryner så saftig at det ikke lages en error report.